circleci/golang-move-project

```diff
@@ -48,4 +48,8 @@ move_project() {
     mkdir $SRC
 }
 
-move_project $HOME/$CIRCLE_PROJECT_REPONAME $HOME/.go_workspace/src/github.com/$CIRCLE_PROJECT_USERNAME/$CIRCLE_PROJECT_REPONAME
```

Comment:
nit: 

- add comments to describe why you might use each of these 
- update the Usage at the top of the file